solana/energy-trading/tests/energy-injection.ts

```diff
@@ -0,0 +1,212 @@
+import * as anchor from "@project-serum/anchor";
+import { Program } from "@project-serum/anchor";
+import { PublicKey } from '@solana/web3.js';
+import { EnergyInjection } from "../target/types/energy_injection";
+import { expect } from 'chai';
+import chai from "chai";
+import chaiAsPromised from "chai-as-promised";
+
+chai.use(chaiAsPromised);
```

Comment:
What is this?

Reply:
Ye hele web3 er basically promise-based, så vi extender chai med en extension der klarer promises